contracts/lib/QuoteV3Auth/V3Struct.sol

```diff
@@ -33,6 +33,11 @@ library V3Struct {
         bytes data;
     }
 
+    struct ParsedQEAuthData {
```

Comment:
This structure is redundant. We can simply store the data size as a uint16 in the original QEAuthData struct. The gas conversion from uint256 to uint16 is negligible.

Reply:
Thanks! NP, I will change this soon. I did not change previous struct is to avoid change code logic as I want to compare the result with exactly the previous path.

another question is: is that safe to change TcbInfo member from uint256 to uint8? https://github.com/automata-network/automata-dcap-v3-attestation/blob/e731c5b532e3b080daf333bac3af405ab20ea30e/contracts/lib/TCBInfoStruct.sol#L11-L15.
As I get response from intel, it's less equal than 255.

Reply:
I wouldn't worry about this for now, as long as it doesn't affect functionality (not breaking) :)